src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -0,0 +1,188 @@
+        dpa_3 = json.load(
+          open(os.path.join('testcases', 'testdata', 'dpa_2.json')))
+
+        # Registration configuration for iteration 1
+        cbsd_records_iter_1 = {'registrationRequests': [device_1, device_2],
+                               'grantRequests': [grant_request_1, grant_request_2],
+                               'conditionalRequests': conditionals_iter_1
+                               }
+        cbsd_records_iter_2 = {'registrationRequests': [device_3, device_4],
+                               'grantRequests': [grant_request_3, grant_request_4]}
+        pal_and_ppa_record_1 = {'PAL': pal_record_1, 'PPA': ppa_record_1}
+        protected_entities_iter_1 = {'PalAndPpa': [pal_and_ppa_record_1], 'ESC': [esc_record_1]}
+        protected_entities_iter_2 = {'GWPZ': [gwpz_record_1], 'FSS': [fss_record_1]}
+
+        # SAS Test Harness FAD records
+        # This section parameters are subjected to change based 
+        # on the SAS Test Harness design, which is currenlty is in progress
+        sas_th_cbsd_0 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_0.json')))
+        sas_th_cbsd_1 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_1.json')))
+        sas_th_cbsd_2 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_2.json')))
+        sas_th_cbsd_3 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_3.json')))
+        sas_th_cbsd_4 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_4.json')))
+        sas_th_cbsd_5 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_5.json')))
+        sas_th_0_cbsd_iter_0 = [sas_th_cbsd_0]
+        sas_th_0_cbsd_iter_1 = [sas_th_cbsd_1]
+        sas_th_0_cbsd_iter_2 = [sas_th_cbsd_2]
+        sas_th_1_cbsd_iter_0 = [sas_th_cbsd_3]
+        sas_th_1_cbsd_iter_1 = [sas_th_cbsd_4]
+        sas_th_1_cbsd_iter_2 = [sas_th_cbsd_5]
+    
+        sas_harness_0 = {
+                    'name': 'SAS-TH-1',
+                    'url': 'localhost:9001',
+                    'certFile': 'server_a.cert',
+                    'keyFile': 'server_a.key',
+                    'certificateHash': 'will be provided in future',
+                    'cbsdRecord': [sas_th_0_cbsd_iter_0, sas_th_0_cbsd_iter_1, sas_th_0_cbsd_iter_2],
+                    'fileServePath': 'sas_harness_2/'
+                }
+        sas_harness_1 = {
+                    'name': 'SAS-TH-2',
+                    'url': 'localhost:9002',
+                    'certFile': 'server_b.cert',
+                    'keyFile': 'server_b.key',
+                    'certificateHash': 'will be provided in future',
+                    'cbsdRecord': [sas_th_1_cbsd_iter_0, sas_th_1_cbsd_iter_1, sas_th_1_cbsd_iter_2],
+                    'fileServePath': 'sas_harness_1/'
+                }
+        # Create the actual config.
+        iter1_config = {
```

Comment:
This config also needs to contain the SAS TH configs (CBSD, PPA, and ESC records) for this iteration. The general SAS TH config (URL, etc.) will be discussed in #368 but I think this PR needs to determine how to specify the per-iteration data that each SAS TH will contain.

Reply:
updated   the sas TH data  with ppa and esc   records